package.json

```diff
@@ -29,15 +34,18 @@
     "lint": "npm run eslint && npm run build && npm run doc && npm run markdownlint",
     "doc": "node utils/doclint/cli.js",
     "clean-lib": "rimraf lib",
-    "build": "npm run tsc && npm run generate-d-ts",
-    "tsc": "npm run clean-lib && tsc --version && npm run tsc-cjs && npm run tsc-esm",
+    "build": "npm run tsc && npm run generate-d-ts && npm run generate-pkg-json",
+    "tsc": "npm run clean-lib && tsc --version && (npm run tsc-cjs & npm run tsc-esm) && (npm run tsc-compat-cjs & npm run tsc-compat-esm)",
```

Comment:
would `&` propagate the error codes if the compilation fails?

Reply:
Yes

Reply:
Is it possible to that the output from both commands would interleave/duplicate?

Reply:
Yes, but I don't believe it will be significant. Since a majority of the time the code compiles fine, this speeds up build times.

Reply:
Yep, makes sense! Thanks for clarifying.